x-pack/plugins/threat_intelligence/public/common/mocks/mock_kibana_data_service.tsx

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { Observable } from 'rxjs';
+import { TimeRangeBounds } from '@kbn/data-plugin/common';
+import * as hook from '../../hooks/use_kibana';
+
+export interface MockSearchServiceParams {
+  searchSubject?: Observable<unknown>;
+  calculateSubject?: TimeRangeBounds;
+}
+
+export const mockKibanaDataService = ({
```

Comment:
I renamed `mockKibanaSearchService` to `mockKibanaDataService` because it now also includes some logic for the `query` section